src/qt/qrimagewidget.cpp

```diff
@@ -92,7 +92,11 @@ bool QRImageWidget::setQR(const QString& data, const QString& text)
 
 QImage QRImageWidget::exportImage()
 {
-    return GUIUtil::GetImage(this);
+    if (!GUIUtil::HasPixmap(this)) {
+        return QImage();
+    }
+
+    return this->pixmap(Qt::ReturnByValue).toImage();
```

Comment:
Micro-Nit:

As far as I understand this is deprecated in favor of `QLabel::pixmap()`

```suggestion
    return this->pixmap().toImage();
```